src/System.cc

```diff
@@ -1401,6 +1401,14 @@ void System::InsertTrackTime(double& time)
 }
 #endif
 
+// NOTE: might not be save to call while orbslam is mapping.
+void System::SaveAtlasAsOsaWithTimestamp(string pathSaveFileName) {
+    string strVocabularyChecksum = CalculateCheckSum(mStrVocabularyFilePath,TEXT_FILE);
+    std::size_t found = mStrVocabularyFilePath.find_last_of("/\\");
```

Comment:
[nit] I believe the std:: here is unnecessary as you are already in the std namespace but i could be mistaken

Reply:
I don't think I'm in the std namespace, since there's no `using namespace std;` and many other commands use `std::`. But maybe I misunderstand?